src/map/utils/battleutils.cpp

```diff
@@ -3667,8 +3667,8 @@ namespace battleutils
 
     bool IsIntimidated(CBattleEntity* PAttacker, CBattleEntity* PDefender)
     {
-        // cannot intimidate yourself!
-        if (PAttacker == PDefender)
+        // cannot intimidate spells targeting the same alligiance!
+        if (PAttacker->allegiance == PDefender->allegiance)
```

Comment:
Does this check break Dragon Affinity intimidate? i.e. rdm wearing crimson casting buffs/cures on a DRG intimidated in era (and it might still?)

Reply:
> Does this check break Dragon Affinity intimidate? i.e. rdm wearing crimson casting buffs/cures on a DRG intimidated in era (and it might still?)

Yeah potentially, good catch, I updated PR to remove that check and replace with the old one as I think for Gration (the only mob that applies status effect based intimidation) it is not needed as will be caught by the ID check `PIntimidateEffect->GetSubID() == PDefender->id`.